Cordova.podspec

```diff
@@ -0,0 +1,37 @@
+require "json"
+
+packageJson = JSON.parse(File.read(File.join(__dir__, "package.json")))
+relVersion = package['version']
+relSource  = { :git => "https://github.com/apache/cordova-ios.git",
+            :tag => "rel/{relVersion}",
+            :submodules => true }
+
+Pod::Spec.new do |s|
+  s.name         = "Cordova"
+  s.version      = relVersion
+  s.summary      = "Apache Cordova for iOS"
+  s.homepage     = "https://github.com/apache/cordova-ios"
+  s.license      = { :type => "Apache 2.0", :file => "LICENSE" }
+  s.author       = { "Apache Software Foundation" }
+  s.platform     = :ios, "11.0"
+  s.source       = relSource
+  s.requires_arc = true
+  s.preserve_paths = 'CordovaLib/cordova.js', 'CordovaLib/VERSION'
+  s.frameworks = 'AssetsLibrary', 'MobileCoreServices', 'AVFoundation', 'CoreLocation'
+  s.default_subspec  = 'Cordova'
+  s.subspec 'Cordova' do |cordova|
+      cordova.source_files = 'CordovaLib/Classes/**/*.{h,m}', 'CordovaLib/Cordova/Cordova.h'
+      cordova.public_header_files = 'CordovaLib/Classes/Public/CDV.h', 'CordovaLib/Classes/Public/CDVAppDelegate.h', 'CordovaLib/Classes/Public/CDVAvailability.h', 'CordovaLib/Classes/Public/CDVAvailabilityDeprecated.h', 'CordovaLib/Classes/Public/CDVCommandDelegate.h', 'CordovaLib/Classes/Public/CDVCommandDelegateImpl.h', 'CordovaLib/Classes/Public/CDVCommandQueue.h', 'CordovaLib/Classes/Public/CDVConfigParser.h', 'CordovaLib/Classes/Public/CDVInvokedUrlCommand.h', 'CordovaLib/Classes/Public/CDVPlugin+Resources.h', 'CordovaLib/Classes/Public/CDVPlugin.h', 'CordovaLib/Classes/Public/CDVPluginResult.h', 'CordovaLib/Classes/Public/CDVScreenOrientationDelegate.h', 'CordovaLib/Classes/Public/CDVTimer.h', 'CordovaLib/Classes/Public/CDVUserAgentUtil.h', 'CordovaLib/Classes/Public/CDVViewController.h', 'CordovaLib/Classes/Public/CDVWebViewEngineProtocol.h', 'CordovaLib/Classes/Private/Plugins/CDVWebViewEngine/CDVWebViewUIDelegate.h', 'CordovaLib/Classes/Public/CDVWhitelist.h', 'CordovaLib/Cordova/Cordova.h', 'CordovaLib/Classes/Public/NSDictionary+CordovaPreferences.h', 'CordovaLib/Classes/Public/NSMutableArray+QueueAdditions.h'
```

Comment:
Can the `public_header_files` property also be written in the same pattern like `source_files`?

```
cordova.public_header_files = 'CordovaLib/Classes/Public/**/*.{h}', 'CordovaLib/Cordova/Cordova.h'
```

I just noticed that this list is manually maintained and that the list is now not matching what is in master.
* `CDVUserAgentUtil.h` had been removed
* `CDVURLSchemeHandler.h` has been added

Reply:
@erisu No - public headers have to be explicitly listed. However, I added a script to automatically update this list every time a change is made. This list is not manually maintained - it's updated whenever `CordovaLib` is built in `Xcode`. It was behind only because this branch is behind `master`. I updated it to merge the latest from `master`, and the update podspec is in [this commit](https://github.com/apache/cordova-ios/pull/543/commits/8b9399e15f944d8f6b17a0c28b3a7273ec6b6f84). Basically, it will be updated automatically as part of the build process once this pull request is merged to `master`.